### ISSUE TITLE
Set-cookie

### ISSUE THREAD

**Issue by JStyle21** [i18next/i18next-http-middleware#49]
Hi,

Can't figure out if there is a set cookie functionality or not, what does the following actually do?

```
  // optional expire and domain for set cookie
  cookieExpirationDate: new Date(),
  cookieDomain: 'myDomain',
  cookiePath: '/my/path',
  cookieSecure: true, // if need secure cookie
  cookieSameSite: 'strict' // 'strict', 'lax' or 'none'
```

**Comment by JStyle21** [comment]
Yes i saw that but what does it actually mean? can i set a cookie with the middleware or does the detector set a cookie? and if so why does a detector need a cookie?

**Comment by adrai** [comment]
It is not mandatory.
But if the language detector is used, it can detect the language, based on various sources, and then "cache" that language for example in the response via cookie header, this can then be used again on client side, etc...

**Comment by JStyle21** [comment]
Ok, so if i choose to cache language then it creates a cookie for that and those are the options for that cookie?

**Comment by adrai** [comment]
yes

**Comment by JStyle21** [comment]
Thank you!

Maybe it will be a good idea to add that part to the docs, something like
// cache user language using a cookie

The current isn't that clear about it
  // cache user language
  caches: false, // ['cookie']

I thought that was a var name for cache but not that "cache = cookie", it's obvious now but not for an outsider.
